TrafficCapture/dockerSolution/src/main/docker/migrationConsole/setupIntegTests.sh

```diff
@@ -43,6 +46,9 @@ fi
 source .venv/bin/activate
 pip install -r requirements.txt
 echo "Starting python 'tests.py'"
-# TODO command to be updated as 'tests.py' gets updated to allow AWS testing
-pytest tests.py
+# TODO Add support to dynamically skip certain tests depending on setup. Currently, forcing negative auth tests not to run here as the source/target cluster has no auth
+# /root/integ-tests/test/reports/.xml
+set -o xtrace
+pytest tests.py::E2ETests::test_0001_index tests.py::E2ETests::test_0002_document tests.py::E2ETests::test_0005_invalidIncorrectUri tests.py::E2ETests::test_0006_OSB --proxy_endpoint="${proxy_endpoint}" --source_endpoint="${source_endpoint}" --target_endpoint="${MIGRATION_DOMAIN_ENDPOINT}" --source_auth_type="none" --target_auth_type="none" --source_verify_ssl=False --target_verify_ssl=False --deployment_type="cloud" --verbose --junitxml="./reports/${unique_id}.xml"
```

Comment:
I was expecting to see the `--unique-id` parameter being used here.

Reply:
You're right. I started added some stuff for unique-id but forgot a couple places here, have updated now